KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyBalanceView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  KarhooLoyaltyBalanceView.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 17.01.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class KarhooLoyaltyBalanceView: UIView {
+    private var balance: Int = 0
+    
+    // MARK: - UI
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.accessibilityIdentifier = KHLoyaltyViewID.balanceLabel
+        label.text = getTitle()
+        label.textAlignment = .center
+        label.font = KarhooUI.fonts.captionBold()
+        label.accessibilityLabel = getTitle()
+        return label
+    }()
+    
+    // MARK: - Init
+    init() {
+        super.init(frame: .zero)
+        self.setupView()
+    }
+    
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupView() {
+        translatesAutoresizingMaskIntoConstraints = false
+        accessibilityIdentifier = KHLoyaltyViewID.balanceView
+        layer.cornerRadius = UIConstants.CornerRadius.medium
+        anchor(height: UIConstants.Dimension.View.largeTagHeight)
+        addSubview(titleLabel)
+        
+        titleLabel.anchor(top: self.topAnchor, leading: self.leadingAnchor, bottom: self.bottomAnchor, trailing: self.trailingAnchor,
```

Comment:
As below: multiline paraphrases wrapping preferred.

https://github.com/karhoo/karhoo-ios-ui-sdk/pull/233/files?file-filters%5B%5D=.swift&show-viewed-files=true#r787055914

Reply:
Mostly resolved. See my comment on where to introduce the line break.